.gitignore

```diff
@@ -11,6 +11,7 @@ Test*FAIL*
 /bots
 test/common/
 test/images/
+/test/static-code
 pkg
```

Comment:
I wonder if you also want to fix up the format of all of the surrounding things like → `pkg` → `/pkg/`

Reply:
Done, according to podman's gitignore. This needs a fix in starter-kit.